<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1212</width>
    <height>803</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(204, 204, 204);
</string>
  </property>
  <widget class="QPushButton" name="b_quit">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>40</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:qlineargradient(spread:pad, x1:0, y1:0, x2:0.99, y2:0.375, stop:0 rgba(42, 98, 116, 255), stop:1 rgba(232, 255, 231, 255));</string>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_preview">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>151</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(125, 105, 124, 255), stop:1 rgba(232, 255, 231, 255));</string>
   </property>
   <property name="text">
    <string>Preview Data</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_search">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>151</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:qlineargradient(spread:pad, x1:0, y1:0, x2:0.99, y2:0.375, stop:0 rgba(42, 98, 116, 255), stop:1 rgba(232, 255, 231, 255))</string>
   </property>
   <property name="text">
    <string>Search For &quot;Chief&quot;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_clean">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>330</y>
     <width>151</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:qlineargradient(spread:pad, x1:0, y1:0, x2:0.99, y2:0.375, stop:0 rgba(42, 98, 116, 255), stop:1 rgba(232, 255, 231, 255))</string>
   </property>
   <property name="text">
    <string>Clean Data</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_two">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>151</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:qlineargradient(spread:pad, x1:0, y1:0, x2:0.99, y2:0.375, stop:0 rgba(42, 98, 116, 255), stop:1 rgba(232, 255, 231, 255))</string>
   </property>
   <property name="text">
    <string>Get Two Columns</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_high">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>360</y>
     <width>151</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:qlineargradient(spread:pad, x1:0, y1:0, x2:0.99, y2:0.375, stop:0 rgba(42, 98, 116, 255), stop:1 rgba(232, 255, 231, 255))</string>
   </property>
   <property name="text">
    <string>Get High Earners</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>430</y>
     <width>151</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:qlineargradient(spread:pad, x1:0, y1:0, x2:0.99, y2:0.375, stop:0 rgba(42, 98, 116, 255), stop:1 rgba(232, 255, 231, 255))</string>
   </property>
   <property name="text">
    <string>Police Employees</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_manager">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>470</y>
     <width>151</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:qlineargradient(spread:pad, x1:0, y1:0, x2:0.99, y2:0.375, stop:0 rgba(42, 98, 116, 255), stop:1 rgba(232, 255, 231, 255));
font:10px;
font-weight:bold;</string>
   </property>
   <property name="text">
    <string>Add Is_Manager Column</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_summary">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>510</y>
     <width>151</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:qlineargradient(spread:pad, x1:0, y1:0, x2:0.99, y2:0.375, stop:0 rgba(42, 98, 116, 255), stop:1 rgba(232, 255, 231, 255));
font-weight:bold;</string>
   </property>
   <property name="text">
    <string>Summary Statistics</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_groupby">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>550</y>
     <width>151</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:qlineargradient(spread:pad, x1:0, y1:0, x2:0.99, y2:0.375, stop:0 rgba(42, 98, 116, 255), stop:1 rgba(232, 255, 231, 255));</string>
   </property>
   <property name="text">
    <string>Group By Year</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_merge">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>590</y>
     <width>151</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:qlineargradient(spread:pad, x1:0, y1:0, x2:0.99, y2:0.375, stop:0 rgba(42, 98, 116, 255), stop:1 rgba(232, 255, 231, 255));</string>
   </property>
   <property name="text">
    <string>Merge Agency Codes</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_9">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>390</y>
     <width>151</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:qlineargradient(spread:pad, x1:0, y1:0, x2:0.99, y2:0.375, stop:0 rgba(42, 98, 116, 255), stop:1 rgba(232, 255, 231, 255))</string>
   </property>
   <property name="text">
    <string>Employees 2013</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>30</y>
     <width>281</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:35px;
background:transparent;</string>
   </property>
   <property name="text">
    <string>Data Analysis</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tw">
   <property name="geometry">
    <rect>
     <x>175</x>
     <y>151</y>
     <width>1031</width>
     <height>301</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white;</string>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>86</number>
   </attribute>
   <column>
    <property name="text">
     <string>Id</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Employee Name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>JobTitle</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Base Pay</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Overtime Pay</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Other Pay</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Benefits</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Total Pay</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Total Pay Benefits</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Year</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Notes</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Status</string>
    </property>
   </column>
  </widget>
  <widget class="QListWidget" name="lw">
   <property name="geometry">
    <rect>
     <x>525</x>
     <y>461</y>
     <width>651</width>
     <height>331</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_lines">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>160</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:transparent;</string>
   </property>
   <property name="text">
    <string>Number of lines</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
